pkg/vm/engine/disttae/logtailreplay/partition_state.go

```diff
@@ -443,6 +443,12 @@ func (p *PartitionState) HandleMetadataDelete(ctx context.Context, input *api.Ba
 	})
 }
 
+func (p *PartitionState) AddBlockId(info catalog.BlockInfo) {
```

Comment:
Don't add public methods if it's for testing only.
Use Handle___ functions instead.